.travis.yml

```diff
@@ -1,10 +1,7 @@
 sudo: false
 language: node_js
-node_js:
-- 4
 
 script:
-- gulp build
```

Comment:
Let's keep the `gulp` process, it generates the browser build that I (and maybe others) used to debug/benchmark.

Reply:
Added it back